app/src/common/shared/org/mozilla/vrbrowser/ui/NavigationBarWidget.java

```diff
@@ -386,21 +322,16 @@ private void exitResizeMode(boolean aCommitChanges) {
         }
         mIsResizing = false;
         mWidgetManager.finishWidgetResize(mBrowserWidget);
-        AnimationHelper.fadeIn(mFocusModeContainer, AnimationHelper.FADE_ANIMATION_DURATION, null);
+        AnimationHelper.fadeIn(mNavigationContainer, AnimationHelper.FADE_ANIMATION_DURATION, null);
```

Comment:
this animation feels a bit janky; if you agree, can we file a follow-up enhancement issue?

Reply:
Not sure what you are referring, can you post a gif/video? do you mean that there is some glitch or is too fast/slow?

Reply:
https://drive.google.com/file/d/17qL5JWkOLyLi6oJg_YuS3zUcii9-MmMH/view?usp=sharing

most noticeable at 00:00:04 and 00:00:09

(I could be commenting in the wrong area of the code, but hopefully you get what I mean)

Reply:
Thanks! I see what you mean, I've added a patch to remove that effect in the latest update, if you try the branch now it should be fixed, otherwise let me know.